GMnetENGINE.gmx/objects/obj_htme.object.gmx

```diff
@@ -49,14 +49,8 @@ htme_init();
         <arguments>
           <argument>
             <kind>1</kind>
-            <string>///Disconnect or kick clients if started
-if (!self.isConnected) exit;
-
-if (self.isServer) {
-    htme_serverShutdown();
-} else {
-    htme_clientDisconnect();
-}
+            <string>/// udphp destroyed us we must clean some now

```

Comment:
This will be called when udphp destroys the obj_htme. And we must do some cleaning then.